libs/core/datastructures/include/hpx/datastructures/detail/small_vector.hpp

```diff
@@ -1273,3 +1427,12 @@ namespace std {    //-V1061
         return num_removed;
     }
 }    // namespace std
+
+namespace hpx::experimental {
+    template <typename T, std::size_t N, typename Alloc, bool emul>
+    struct is_trivially_relocatable<
+        hpx::detail::small_vector<T, N, Alloc, emul>>
+      : is_trivially_relocatable<T>
```

Comment:
And `Alloc` needs to be trivially relocatable, and `Alloc::pointer`, and it must have the right POCMA/POCCA/POCS behavior if you care about any of that stuff. See https://github.com/Quuxplusone/llvm-project/blob/trivially-relocatable-v107/libcxx/include/__memory/allocator_traits.h#L421-L433 and https://github.com/Quuxplusone/llvm-project/blob/trivially-relocatable-v107/libcxx/include/vector#L399-L407 for my approach.

Reply:
Ah, that is interesting. Will correct and write tests for this.